### ISSUE TITLE
Plot Hole Image w/ Shots

### ISSUE THREAD

**Issue by quinnpertuit** [rer145/research-project-mu#1]
Have you worked through plotting the shot-by-shot over the image at scale with hole location?

**Comment by rer145** [comment]
No, I have not. This was a project for grad school and I graduated May 2019. I haven't had time to continue with it further. Is there a certain step in the process you're having issues with?
